Resources/Prototypes/_NF/Entities/Objects/Consumable/Food/produce.yml

```
@@ -410,3 +410,35 @@
   - type: DeepFrySpawn
     cycles: 2
     output: FoodOnionRings
+
+- type: entity
+  name: monkana
+  parent: FoodProduceBase
+  id: FoodMonkana
+  description: Looks like a monkeyfied banana.
+  components:
+  - type: FlavorProfile
+    flavors:
+      - monkana
```

```suggestion
  name: monkanana
  parent: FoodProduceBase
  id: FoodMonkanana
  description: Looks like a monkeyfied banana.
  components:
  - type: FlavorProfile
    flavors:
      - monkanana
```
Consistency in naming.